bundles/org.openhab.binding.lghombot/src/main/java/org/openhab/binding/lghombot/internal/CameraUtil.java

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.lghombot.internal;
+
+import java.awt.image.BufferedImage;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+
+import javax.imageio.ImageIO;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.smarthome.core.library.types.RawType;
+import org.eclipse.smarthome.core.types.State;
+import org.eclipse.smarthome.core.types.UnDefType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link CameraUtil} is responsible for parsing the raw yuv 422 image from the LG HomBot.
+ *
+ * @author Fredrik Ahlström - Initial contribution
+ */
+@NonNullByDefault
+public class CameraUtil {
+
+    private static final Logger logger = LoggerFactory.getLogger(CameraUtil.class);
+
+    private CameraUtil() {
+        // No need to instance this class.
+    }
+
+    static State parseImageFromBytes(byte[] yuvData) {
+
+        final int width = 320;
```

Comment:
These could be real class level constants

Reply:
I added the width and height as arguments instead since it should be pretty reusable, but might need pixel/row modulo to work for all cases.